jupyter-on-gke/variables.tf

```diff
@@ -23,3 +23,47 @@ variable "create_namespace" {
   description = "Enable creation of jupyterhub namespace if it does not exist"
   default     = false
 }
+
+variable "add_auth" {
+  type = bool
+  description = "Enable iap authentication on jupyterhub"
+  default = false
```

Comment:
default true?

Reply:
I will send another PR once the autopilot part is figured out.